resources/rules/aws-network-and-content-delivery-cloudfront-origin-failover-enabled.yaml

```diff
@@ -0,0 +1,14 @@
+id: 1fb62a14-da79-4bdd-b199-1f9868004a38
+refId: aws-network-and-content-delivery-cloudfront-origin-failover-enabled.yaml
+type: asset
+name: >
+  Checks whether an origin group is configured for the distribution of at least 2 origins in the origin group for Amazon CloudFront.
+description: >
+  Checks whether an origin group is configured for the distribution of at least 2 origins in the origin group for Amazon CloudFront. This rule is NON_COMPLIANT if there are no origin groups for the distribution.
+enabled: true
+sql: >
+  SELECT arn as assetid
+  FROM ${magpie_schema}.awscloudfrontdistribution
+  WHERE (configuration -> 'originGroups' -> 'members' -> 'quantity' < 2)
```

Comment:
WHERE (configuration -> 'originGroups' -> 'members' -> 'quantity' < 2) causes this error

ERROR | i.o.p.s.OpenRavenManagedCrudService | Error saving rule 1fb62a14-da79-4bdd-b199-1f9868004a38. Details: SQL ERROR: operator does not exist: jsonb < integer
  Hint: No operator matches the given name and argument types. You might need to add explicit type casts.